Procfile.dev

```diff
@@ -1,3 +1,3 @@
-web: bin/rails server -b 0.0.0.0 -p 3000
+web: rm -f tmp/pids/server.pid & bin/rails server -b 0.0.0.0 -p 3000
```

Comment:
Can you move this to the dockerfile instead?   This pid check is useful locally.

Reply:
I think you can do `     PIDFILE: /dev/null` in the environment variable section of docker-compose.yml

Reply:
I did try to put it there at first, but it didn't work as:

    RUN rm -f /app/tmp/pids/server.pid

I guess because that just runs once when the container is being built! Maybe if I put it into the entry point it will take though? I'll give that a try.

Reply:
`PIDFILE: /dev/null` worked nicely!